src/app/_services/api.service.ts

```diff
@@ -69,13 +71,14 @@ export class ApiService {
       'grid_data': JSON.stringify(this.feature.gridData),
       'quoted': this.feature.quoted,
       'archived': this.feature.archived,
-      'quantity': this.feature.quantity
+      'quantity': this.feature.quantity,
+      'is_quantity_order': this.feature.is_quantity_order
     };
 
-    // const headers = new HttpHeaders({'Content-Type': 'application/json'});
-    // const options = new RequestOptions({headers: headers});
+    console.log('patchData:', patchData);
```

Comment:
should use `this.debug.log` so that we don't have console logs break a browser on live that may not have console, avoid logging slowing down the app, and for consistency.